packages/tupaia-web/src/api/mutations/useRequestResetPassword.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Tupaia
+ *  Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
+ */
+
+import { useMutation, useQueryClient } from 'react-query';
+import { post } from '../api';
+
+type ResetPasswordParams = {
+  emailAddress: string;
+};
+export const useRequestResetPassword = () => {
+  const queryClient = useQueryClient();
+
+  return useMutation<any, Error, ResetPasswordParams, unknown>(
+    ({ emailAddress }: ResetPasswordParams) => {
+      return post('resetPassword', {
+        data: {
+          emailAddress,
+        },
+      });
+    },
+    {
+      onSuccess: () => {
+        queryClient.invalidateQueries();
+      },
+    },
```

Comment:
```suggestion
   
```
The invalidate queries cleares all the react query caches. We need it when we login and logout as the user's permissions determine which data they get from the api but we don't need it here as the user stays logged out